assets/stylesheets/style.scss

```diff
@@ -467,6 +467,18 @@
 	}
 }
 
+.wcs-pointer-page-dimmer {
+	display: none;
+	position: absolute;
```

Comment:
Right now, with `position: absolute;` the background doesn't extend to the bottom of the page if the user scrolls down. You can fix it by using these properties instead:

``` css
position: fixed;
top: 0;
left: 0;
bottom: 0;
right: 0;
```

Works in Chrome, and pretty sure it will work in IE, but needs to be tested there.